src/test/java/bio/terra/service/filedata/google/firestore/FireStoreFileDaoTest.java

```diff
@@ -109,6 +112,29 @@ public void createDeleteFileTest() throws Exception {
     assertFalse("File doesn't exist after delete", fileExisted);
   }
 
+  @Test
+  public void listAllFilesTest() throws Exception {
+    List<FireStoreFile> noFiles = fileDao.enumerateFiles(firestore, collectionId, 0, 10);
+    assertEquals(noFiles.size(), 0);
+
+    List<FireStoreFile> fileList = IntStream.range(0, 5).boxed().map(i -> makeFile()).toList();
+    fileDao.upsertFileMetadata(firestore, datasetId, fileList);
+    List<String> fileIds = fileList.stream().map(FireStoreFile::getFileId).toList();
+    for (String fileId : fileIds) {
+      FireStoreFile existCheck = fileDao.retrieveFileMetadata(firestore, datasetId, fileId);
+      assertNotNull("File entry was created", existCheck);
+    }
+
+    List<FireStoreFile> allFiles = fileDao.enumerateFiles(firestore, collectionId, 0, 10);
+    assertEquals(allFiles.size(), 5);
```

Comment:
Here I would use `assertThat`
```suggestion
    assertThat(allFiles, hasSize(5));
```